### PR TITLE
CrystEngComm Revision

### PR DESCRIPTION
**Pull request by kbsezginel**
- Moved cif files into a new repository
- Added setup.py
- Organized and updated jupyter notebooks
- Updated README

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,3 +12,93 @@ launch_workers_qsub.sh~
...
+.hypothesis/
+
+# Translations
+*.mo
+*.pot
+
+# Django stuff:
+*.log
+local_settings.py
+
```
Comment (paulboone) [inline .gitignore:70]: this is pretty "comprehensive"... is there a django application in here somewhere? ;)

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import os
+from setuptools import setup, find_packages
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+
```
Comment (paulboone) [inline setup.py:6]: This works here, but will not work in general so wouldn't recommend this as a practice.

**Review by paulboone — APPROVED** [review]
You'll want to change how you're loading requirements in the setup script, but you can do that in a future commit.

File: .gitignore (hunk 1 above)
Comment (kbsezginel) [reply .gitignore:70]: No :) This is the default gitignore that github provides when you select python as the language. I should probably clean it up.

File: setup.py (hunk 2 above)
Comment (kbsezginel) [reply setup.py:6]: You are right I realized it does not work with github repo links for example. I wasn't sure what would be a better way to do it.

**Pull request merged**